conversation_reconstruction_dataflow/conversation_reconstruction/construct_utils/conversation_constructor.py

```diff
@@ -1,25 +1,28 @@
+
```

Comment:
This file is kind of complicated. Worth making some tests I think.

Reply:
Most parts of the code were checked using actual data previously.
But I'll write some documentation and tests later, will put up a issue for this.